### ISSUE TITLE
Love your logic! and whop whop Waterfall function!

### ISSUE THREAD

**Issue by rebecacalvoquintero** [azayneeva/week-5-YARA#31]
maybe delete the console.log and it will look perfect!! =)

**Comment by BartBucknill** [comment]
I totally agree, this is awesome! You've got loads of great stuff done here, including using an encryption module and process.env to manage api keys etc. This is great, well done guys! 👍
